persistence/sql/migrations/templates/20210913095309_identity_recovery_tokens.down.fizz

```diff
@@ -0,0 +1,2 @@
+change_column("identity_recovery_tokens", "identity_recovery_address_id", "uuid", {"size": 36})
```

Comment:
Without setting all NULL columns to some valid value, or deleting their rows, this will fail.

Reply:
Fixed in https://github.com/ory/kratos/pull/1750/commits/c6f3d98a92cdbb928e105fa1ae653e5796c6a155